src/tests/system/tests/test_sssctl.py

```diff
@@ -67,3 +68,55 @@ def test_sssctl__check_invalid_id_provider(client: Client):
         "[rule/sssd_checks]: Attribute 'id_provider' in section 'domain/test' has an invalid value: invalid"
         in output.stdout_lines[1]
     )
+
+
+@pytest.mark.ticket(bz=1902280)
+@pytest.mark.topology(KnownTopology.LDAP)
+def test_sssctl__reset_cached_timestamps(client: Client, ldap: LDAP):
+    """
+    :title: fix sss_cache to also reset cached timestamp
+    :setup:
+        1. Add user to SSSD
+        2. Add group to SSSD
+        3. Start SSSD
+    :steps:
+        1. Call getent group
+        2. Set domain config
+        3. Restart SSSD
+        4. Call getent group
+        5. Modify group entry in LDAP
+        6. Call 'sss_cache -E'
```

Comment:
should be s/sss_cache/sssctl/ right?

Reply:
Yes, this shloud be `sssctl cache-expire`